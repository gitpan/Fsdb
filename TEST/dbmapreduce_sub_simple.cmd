prog='dbmapreduce'
args='-k experiment -f TEST/dbmapreduce_sub_simple.pl -C "simple_reducer"'
cmp='diff -cb '
in=TEST/dbmapreduce_ex.in
