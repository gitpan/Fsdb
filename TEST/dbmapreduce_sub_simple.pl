#!/usr/bin/perl

sub simple_reducer {
    dbroweval('-m', 
	'-b' => 'my $count = 0; @out_args = (-cols =>[qw(n)]);',
	'-e' => '$ofref = [ $count ];',
	' $count++; ');
}
