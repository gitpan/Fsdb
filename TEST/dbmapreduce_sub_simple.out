#fsdb experiment n
ufs_mab_sys	2
ufs_rcp_real	2
#   | dbmapreduce -k experiment -f TEST/dbmapreduce_sub_simple.pl -C simple_reducer
